### ISSUE TITLE
Delivery Status

### ISSUE THREAD

**Issue by Raymondss12** [vector-im/element-meta#328]
#### What would you like to do?
We need to see if an outgoing message has been delivered
#### Why would you like to do it?
It would be nice to see if your message is delivered

**Comment by robintown** [comment]
Send status is indicated by a check to the right of a message:

![Screenshot 2022-04-05 at 20-01-28 Element Test room](https://user-images.githubusercontent.com/48614497/161870283-264147d0-40f5-4ee9-b0d4-950a96f5decd.png)

**Comment by Raymondss12** [comment]
That shows that your message has been sent. It doesn't show if it has been delivered successfully. E.g Whatsapp shows double checkmark when message is delivered.

**Comment by t3chguy** [comment]
Delivery to what though? From your server to their server? To one of their devices? To all of their devices?

**Comment by Raymondss12** [comment]
Well this depends whether it's for direct messages or rooms. For direct messages you should be able to see when the message you've sent has been delivered on the other user's phone. Basically the double checkmark concept in WhatsApp and signal. For rooms you could have it when you long press the message you sent you can see all users the message has been delivered to.

**Comment by t3chguy** [comment]
Right, but in Matrix a message has multiple levels of delivery

You send a message
-> Delivered to your server
-> Delivered to the recipient's server
-> Delivered to a recipient's device

The Matrix spec only lets us know when its delivered to your server, nothing more

**Comment by Raymondss12** [comment]
Yeah but there is no way to activate "Delivered to a recipient's device". Which is very crucial for many users.

**Comment by t3chguy** [comment]
That's a Matrix spec issue, not an Element one. For a privacy oriented product it's not exactly crucial.

**Comment by ghost** [comment]
Should we file an issue on #matrix-spec?

**Comment by robintown** [comment]
Yes, you can file one there.
